docs/source/concepts/PropertiesFile.rst

```diff
@@ -285,6 +285,8 @@ Plotting Settings
 +---------------------------------+------------------------------------------------------------------+---------------------+
 |``plots.legend.Location``        |Default legend location                                           |``best``             |
 +---------------------------------+------------------------------------------------------------------+---------------------+
+|``plots.images.Colormap``        |Default colormap for image plots                                  |``cool``             |
```

Comment:
Is the value in the right-hand column supposed to match the default in the template above?

Reply:
I didn't think so because the column heading just says "Example value" but I can change it to the default colormap.